package.json

```diff
@@ -20,6 +20,7 @@
   },
   "homepage": "https://github.com/hack-curtains/project-gordon-web#readme",
   "dependencies": {
+    "@babel/core": "^7.16.12",
```

Comment:
Shouldn't babel be in the devDependencies?

Reply:
I don't know if this was due to version difference, but I had to install the another @babel/core because I was getting 'cannot find module babel-core'.

I just deleted in this PR.